repository/SmalltalkCI-Pharo-Coverage-Core.package/SCIPharo13CodeCoverage.class/properties.json

```diff
@@ -0,0 +1,11 @@
+{
+	"commentStamp" : "",
+	"super" : "SCIPharoCodeCoverage",
+	"category" : "SmalltalkCI-Pharo-Coverage-Core",
+	"classinstvars" : [ ],
+	"pools" : [ ],
+	"classvars" : [ ],
+	"instvars" : [ ],
+	"name" : "SCIPharo13CodeCoverage",
```

Comment:
This class does not have any methods, right? Why is it needed or can it be removed?

Reply:
I did not create that class :)

Reply:
That was me. It's not required. I've removed it.